pkgs/tools/text/ugrep/default.nix

```diff
@@ -9,15 +9,15 @@
 , zlib
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
`meta.changelog` also needs to be updated

Reply:
Done. Thanks!